### PR TITLE
set T1 LR in the VSVip of Passthrough VS

### PR DESCRIPTION
**Pull request by monotosh-avi**
Without this change passthrough VSVip creation fails in nsxt environment.
fixes AV-135928

### CODE REVIEW

File: internal/nodes/avi_model_tls_passthrough.go [hunk 1]
```diff
@@ -66,6 +66,10 @@ func (o *AviObjectGraph) BuildVSForPassthrough(vsName, namespace, hostname, key
 		VipNetworks: lib.GetVipNetworkList(),
 	}
 
+	if lib.GetT1LRPath() != "" {
```
Comment (akshayhavile) [inline internal/nodes/avi_model_tls_passthrough.go:69]: @monotosh-avi : At line number 53, We need to set "vrf-context" of passthrough VS node to empty for NSX-T environment.

Comment (monotosh-avi) [reply internal/nodes/avi_model_tls_passthrough.go:69]: good catch. updated.

File: internal/nodes/avi_model_tls_passthrough.go [hunk 2]
```diff
@@ -49,8 +49,11 @@ func (o *AviObjectGraph) BuildVSForPassthrough(vsName, namespace, hostname, key
 	avi_vs_meta.ApplicationProfile = utils.DEFAULT_L4_APP_PROFILE
 	avi_vs_meta.NetworkProfile = utils.DEFAULT_TCP_NW_PROFILE
 
-	vrfcontext := lib.GetVrf()
-	avi_vs_meta.VrfContext = lib.GetVrf()
+	vrfcontext := ""
+	if lib.GetT1LRPath() == "" {
+		vrfcontext = lib.GetVrf()
+		avi_vs_meta.VrfContext = lib.GetVrf()
```
Comment (akshayhavile) [inline internal/nodes/avi_model_tls_passthrough.go:55]: we can use "vrfcontext" instead of calling lib.GetVrf()

**Review by akshayhavile — APPROVED** [review]
LGTM with a minor nit.

**Review by akshayhavile — APPROVED** [review]

**Comment by monotosh-avi** [comment]
build ako

**Comment by monotosh-avi** [comment]
build ako

**Comment by monotosh-avi** [comment]
build ako

**Review by chauhanshubham — APPROVED** [review]

**Pull request merged**